docs/01-get-started/03-golang-hello-world.md

```diff
@@ -174,10 +174,16 @@ func helloWorldActivity(ctx context.Context, name string) (string, error) {
 Don't forget to register the workflow and activity to the worker.
 
 ```go
-func init() {
-    workflow.Register(helloWorldWorkflow)
-    activity.Register(helloWorldActivity)
-}
+// add to the function startWorker before calling worker.Start()
+
+// if err != nil {
```

Comment:
mind removing?

Reply:
@davidporter-id-au Are there any concerns from you about this PR?